src/odin/maneuversbuilder.cc

```diff
@@ -2812,55 +2815,96 @@ uint16_t ManeuversBuilder::GetExpectedTurnLaneDirection(Maneuver& maneuver) cons
 void ManeuversBuilder::ProcessTurnLanes(std::list<Maneuver>& maneuvers) {
   auto curr_man = maneuvers.begin();
   auto next_man = maneuvers.begin();
+  auto next_next_man = maneuvers.begin();
```

Comment:
I need to step through this in more detail. I was thinking after we talked - to use the previous, current, and next maneuver pattern instead of the `next next maneuver`

Reply:
Made that change. Can you take another look?